lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -323,6 +325,18 @@ module InternalConstants
                  :default => nil,
                  :secret => true
 
+    # The URL of Stackdriver Logging API. Right now this only works with the
+    # gRPC path (use_grpc = true).
+    # The default value is: https://logging.googleapis.com.
+    # One common usage of this config is to provide a mocked / stubbed Logging
+    # API. e.g. http://localhost:52000.
+    config_param :logging_api_url, :string, :default => DEFAULT_LOGGING_API_URL
+
+    # Whether to use a secure channel when talking to the Logging API. This
+    # should only be false if the Stackdriver Logging Agent is set up to talk
+    # to a mocked / stubbed Logging API. Otherwise the requests will fail.
+    config_param :use_secure_channel, :bool, :default => true
```

Comment:
Isn't this equivalent to having the `https://` protocol on the `logging_api_url` (as opposed to the `http://` protocol)?

Reply:
What I mean is, should we use the `http://` vs. `https://` prefix to infer this value, rather than adding a separate config option?